.github/CONTRIBUTING.md

```diff
@@ -24,10 +24,10 @@ export default class IgcSomethingComponent extends LitElement {
    * 1. Static members
    */
   /** @private */
```

Comment:
These `private` comments have been removed in the code. Which is the proper expectation - having them or not having them?

Reply:
After the latest `web-component-analyzer` update these comments are not needed.